src/Dev/FixFilePermissionsTask.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SilverStripe\Dev\Tasks;
+
+use SilverStripe\Assets\File;
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\ORM\Queries\SQLUpdate;
+use SilverStripe\Security\InheritedPermissionFlusher;
+
+/**
+ * Files imported from SS3 might end up with broken permissions if there is a case conflict.
```

Comment:
Should mention that this fixes issues with https://github.com/silverstripe/silverstripe-secureassets